test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] You should add this file ( `test.md` ) to the `.gitignore` file so that `git` (version control) doesn't track it and update your GitHub repo.